_pytest/python.py

```diff
@@ -1777,7 +1777,7 @@ def pytest_generate_tests(self, metafunc):
                 if fixturedef.params is not None:
                     func_params = getattr(getattr(metafunc.function, 'parametrize', None), 'args', [[None]])
                     # skip directly parametrized arguments
-                    if argname not in func_params and argname not in func_params[0]:
```

Comment:
I didn't figure out why this line is needed. It has been introduced with https://bitbucket.org/pytest-dev/pytest/pull-requests/257/allow-to-override-parametrized-fixtures

I suppose it's due to strings like ´value,othervalue`, but it doesn't seem like it's actually needed.